src/modules/taskList.js

```diff
@@ -0,0 +1,126 @@
+import Task from './task.js';
+import Status from './status.js';
+
+export default class TaskList {
+  constructor() {
+    this.listArray = [];
+  }
+
+  display() {
+    const listContainer = document.querySelector('.listContainer');
+    if (listContainer) {
+      listContainer.innerHTML = '';
+    }
+    this.listArray.forEach((el) => {
+      const li = [];
+      const input = [];
+      const p = [];
+      const button = [];
+      // Create List Element
+      li[el.index] = document.createElement('li');
+      li[el.index].setAttribute('id', el.index);
+      li[el.index].contentEditable = true;
+
+      // Create Input Element
+      input[el.index] = document.createElement('input');
+      input[el.index].setAttribute('type', 'checkbox');
+      input[el.index].classList.add('checkbox');
+      input[el.index].setAttribute('id', el.index);
+
+      // Create P Element
+      p[el.index] = document.createElement('p');
+      p[el.index].textContent = el.description;
+      p[el.index].setAttribute('id', el.index);
+
+      // Create Button Element
+      button[el.index] = document.createElement('button');
+      button[el.index].setAttribute('id', el.index);
+      button[el.index].innerHTML = '<i class="fa-solid fa-ellipsis-vertical"></i>';
+      li[el.index].append(input[el.index], p[el.index], button[el.index]);
+      listContainer.append(li[el.index]);
+
+      // Edit P Element
+      p[el.index].addEventListener('click', (e) => {
+        e.target.nextSibling.innerHTML = '<i class="fa-solid fa-trash-can"></i>';
+        e.target.nextSibling.style.cursor = 'pointer';
+
+        // Remove List Element
+        e.target.nextSibling.addEventListener('click', () => {
+          li[el.index].remove();
+          this.remove(el.index);
+        });
+      });
```

Comment:
- [ ] [OPTIONAL] It is difficult to delete a task that is not completed because **the trash icon is not easily accessible**. Please make it easily accessible. 💡  You can replace it with that `:` icon on the right side of each task as it is not necessary 💡 😎.